defaults.yml

```diff
@@ -3,15 +3,20 @@ neo4j:
   bolt_uri: ''
   username: ''
   password: ''
+api:
```

Comment:
If the number of workers and files changes the user isn't going to remember to change this, maybe link this to the latest run on the server? Or a grep or wc -l or something on the data directory to find the number of workers and files?